### ISSUE TITLE
[configs] teleop config requires joint_map_path in main YAML

### ISSUE THREAD

**Issue by liesrock** [ADVRHumanoids/xbot2_examples#9]
Running the examples I noticed that inside `config/teleop_simple_example.yaml` the joint_map_path inside the XBotInterface YAML node is missing.

This leads (given the hybrid binary + source for xbot2_wip installation) to this error:

```
[error] in FromConfigFile : XBotInterface node of  /home/liesrock/src/xbot2_examples/config/teleop_simple_example.yaml  does not contain joint_map_path mandatory node!!
terminate called after throwing an instance of 'std::runtime_error'
  what():  Config file parsing failed, see error above
Aborted (core dumped)
```

Is this a problem related to my setup @alaurenzi ?

**Comment by alaurenzi** [comment]
Old binaries for XbotInterface, it will be fixed by re-generating them for 18.04 as well.

**Comment by liesrock** [comment]
ok I will try to generate the new ones and check it: in case I delete the pull request.

**Comment by torydebra** [comment]
for my setup (16) this was fixed #4

**Comment by liesrock** [comment]
Solved with the new binaries (to be found here for a test http://54.194.210.201/repo/amd64/bionic-18.04-2020_09_22_09_52_46.zip)
